ipatests/test_webui/crypto_utils.py

```diff
@@ -63,9 +63,9 @@ def generate_certificate(hostname):
     ).serial_number(
         x509.random_serial_number()
     ).not_valid_before(
-        datetime.utcnow()
+        datetime.now(tz=datetime.UTC)
```

Comment:
As the import is
```
from datetime import datetime, timedelta
```
you need to replace with
```
from datetime import datetime, timedelta, UTC
now = datetime.now(tz=UTC)
```

Reply:
Thanks, fixed.